### ISSUE TITLE
Versions in CustomResourceDefinitions: updating status with kubectl is now possible

### ISSUE THREAD

**Issue by sbueringer** [kubernetes/website#34463]
The doc states:

> Note:
> The kubectl tool currently cannot be used to edit or patch the status
> https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version

Starting with v1.24 kubectl edit can be used to update the status (not sure about patch)

**Comment by Shubham82** [comment]
Hi @sbueringer 
It is implemented in k8sv1.24. Here is the PR which implements this [k8s#99556](https://github.com/kubernetes/kubernetes/pull/99556)

We have to update the `Note` section in the docs.

**Comment by sbueringer** [comment]
Yup thx. I'm aware. Just wanted to open the issue so we know we have to update the note

**Comment by sbueringer** [comment]
I wonder if we should instead replace the current example with one using kubectl patch. (+ removing the language around that kubectl doesn't support updating the status)

**Comment by Shubham82** [comment]
Sorry, I forgot to mention updating the content of the Note.

> I wonder if we should instead replace the current example with one using kubectl patch. (+ removing the language around that kubectl doesn't support updating the status)

Seems good.

**Comment by Shubham82** [comment]
> kubectl patch customresourcedefinitions <CRD_Name> --subresource='status' --type='merge' -p '{"status":{"storedVersions":{"value":v1}}}'

@sbueringer is it valid if we mention above as an example for using the kubectl patch to Update a CRD status through the scale subresource using a merge patch.

**Comment by sbueringer** [comment]
Almost. This should be correct:
```bash
kubectl patch customresourcedefinitions <CRD_Name> --subresource='status' --type='merge' -p '{"status":{"storedVersions":["v1"]}}'
```
(I verified it locally and it works)

**Comment by Shubham82** [comment]
> ```shell
> kubectl patch customresourcedefinitions <CRD_Name> --subresource='status' --type='merge' -p '{"status":{"storedVersions":["v1"]}}'
> ```
yeah, v1 should be in [ ], thanks, @sbueringer  for verifying it.

**Comment by Shubham82** [comment]
I will open a pr for it soon.
/assign

**Comment by sbueringer** [comment]
Thank you! Feel free to assign me for review

**Comment by Shubham82** [comment]
> Thank you! Feel free to assign me for review

Sure

**Comment by Shubham82** [comment]
Fixed it here: #34468
